python/GafferImageUI/CatalogueUI.py

```diff
@@ -210,7 +219,7 @@ def _imageCellData( self, image, catalogue ) :
 
 		value = self.__defaultValue
 
-		metadata = catalogue["out"].metadata()
+		metadata = catalogue["out"].metadata( self._standardViewName( catalogue["out"] ) )
```

Comment:
Rather than fix this manually for every single column (other people have custom columns, and there are columns in Gaffer that aren't fixed in this commit - see `startup/GafferSceneUI/catalogue.py`), we should just set `image:viewName` once at line 109, before delegating to `_imageCellData`.